src/pr-info.ts

```diff
@@ -446,52 +442,24 @@ function analyzeReviews(prInfo: PR_repository_pullRequest, isOwner: (name: strin
     });
 }
 
-function getDangerLevel(categorizedFiles: readonly FileLocation[]) {
+function getDangerLevel(categorizedFiles: readonly FileInfo[]) {
     if (categorizedFiles.some(f => f.kind === "infrastructure")) {
         return "Infrastructure";
+    }
+    const packagesTouched = getPackagesTouched(categorizedFiles);
+    if (packagesTouched.length === 0) { // ????
```

Comment:
If you edit the root `README.md` or localization PRs etc

Reply:
I know -- that `???` comment was there before, I just moved it up.  (It didn't seem questionable to me at all, so I figured that someone had a reason to put it there...)